experimental/packages/opentelemetry-instrumentation-grpc/src/grpc-js/clientUtils.ts

```diff
@@ -23,7 +23,7 @@ import {
   propagation,
   context,
 } from '@opentelemetry/api';
-import type * as grpcJs from '@grpc/grpc-js';
+import { Client, Metadata, ServiceError } from '@grpc/grpc-js';
```

Comment:
Hmm, I'm not too deep into writing instrumentations, but is there a chance that `Metadata` imported here will be of a different `@grpc/grpc-js` version than the one we're currently wrapping? :thinking: 

It's spliced into the arguments when metadata is not provided so that we can propagate context. I'm a bit worried that we may end up with the wrong one there.

Reply:
This package doesn't have a `@grpc/grpc-js` dependency. It can only be imported from the root project I think.

Reply:
Yep, that's my concern.

Suppose we explicitly depend on `@grpc/grpc-js@1.2.6` in the root project. A dependency of that project uses `@grpc/grpc-js@1.8.17`, so they're not de-duped. On `require`, we will each of them using this code, and then we will splice `Metadata` from `@grpc/grpc-js@1.2.6` into calls made with `@grpc/grpc-js@1.8.17` and we may be missing any fixes that were made from `1.2.6` to `1.8.17`.

Looking closer at it, I think this `import` may also cause a crash on runtime if `@opentelemetry/instrumentation-grpc` is instantiated but `@grpc/grpc-js` is installed as a dependency in the user's app. :thinking: 

It may be worth to keep passing through the `grpcClient: typeof grpcJs` arg up to `getMetadata()` to avoid this import. :thinking:

Reply:
Done.